.travis.yml

```diff
@@ -1,6 +1,9 @@
 sudo: required
 dist: trusty
 
+notifications:
+  email: false
```

Comment:
mmm yeah, open an issue on hackgt/hackgproject , maybe we'll make a hackgt bot account that gets all these things?